bin/webpack-helpers.js

```diff
@@ -192,6 +194,12 @@ const getMainConfig = ( options = {} ) => {
 						chunks: 'all',
 						priority: 10,
 					},
+					'vendors-style': {
+						test: /\/node_modules\/.*?style\.s?css$/,
+						name: 'vendors-style',
+						chunks: 'all',
+						priority: 7,
+					},
```

Comment:
what is this configuration picking up?

Reply:
The idea was to have `@wordpress/components` styles (and any future `node_modules` stylesheet) loaded in an independent file named `vendors-style.css`. But maybe it's better to bundle them in the generic `style.css`?

I think it all depends on how big the `vendors-style.css` file will end up being, now it's pretty small but as we keep adding more components from `@wordpress/components`, it will start getting bigger.

Reply:
So is the idea here that there will be some blocks that don't need the vendors-style whereas _all_ blocks use the generic styles?  If so, then let's leave this separation in place for now.  We can always revisit if bundle needs change.